clang/lib/Lex/LiteralSupport.cpp

```diff
@@ -1805,6 +1818,10 @@ CharLiteralParser::CharLiteralParser(const char *begin, const char *end,
     largest_character_for_kind = 0x7Fu;
   }
 
+  llvm::CharSetConverter *Converter = nullptr;
+  if (!isUTFLiteral(Kind) && LiteralConv)
+    Converter = LiteralConv->getConverter(ToExecCharset);
+
```

Comment:
Lets not convert wide strings at all for now (and not unevaluated string either)

Reply:
Sure, I removed wide character translation for now, thanks!